src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -384,6 +398,54 @@ def _resolve_http_error_count(
             alias,
         )
 
+    def _resolve_percentile_percent_change(
+        self,
+        args: Mapping[str, Union[str, Column, SelectType, int, float]],
+        alias: Optional[str] = None,
+    ) -> SelectType:
+        if self.builder.start is None or self.builder.end is None:
+            raise InvalidSearchQuery("Need both start & end to use percentile_percent_change")
+        middle = self.builder.start + (self.builder.end - self.builder.start) / 2
+        first_half = function_aliases.resolve_metrics_percentile(
+            args=args,
+            alias=None,
+            fixed_percentile=args["percentile"],
+            extra_conditions=[
+                Function(
+                    "less",
+                    [
+                        Function("toDateTime", [middle]),
+                        self.builder.column("timestamp"),
+                    ],
+                ),
+            ],
+        )
+        second_half = function_aliases.resolve_metrics_percentile(
+            args=args,
+            alias=None,
+            fixed_percentile=args["percentile"],
+            extra_conditions=[
+                Function(
+                    "greaterOrEquals",
+                    [
+                        Function("toDateTime", [middle]),
+                        self.builder.column("timestamp"),
+                    ],
+                ),
+            ],
+        )
+        return Function(
+            "multiply",
+            [
+                Function(
+                    "divide",
+                    [second_half, first_half],
+                ),
+                100,
```

Comment:
we don't normally do the multiply 100 on any of our rates right? we just return the ratio and convert to percentage in the fe